mk/mockery.mk

```diff
@@ -0,0 +1,11 @@
+MOCKERY_VERSION := 2.36.1
```

Comment:
I think this is okay as is, but we will likely forget to update this as time goes on.  you could capture that with:

curl -L https://api.github.com/repos/vektra/mockery/releases/latest | jq --raw-output .tag_name

to get something like  v2.36.1
and then you'd just need to trim off the 'v' for the URL below.  Thoughts

Reply:
i think that would be a good addition. so then i'd need to add a check to determine which version is installed and update the target if it isn't the latest?

Reply:
nah, i think you could just rely on a 'make clean'  which should clear it out

Reply:
meaning, fresh setups would get the newest, and people could easily upgrade as needed